gguf-py/gguf/gguf.py

```diff
@@ -723,6 +754,84 @@ def add_pad_token_id(self, id: int):
         self.add_uint32(KEY_TOKENIZER_PAD_ID, id)
 
 
+class SpecialVocab:
```

Comment:
Not all files in added_tokens.json are special tokens, so this name is confusing.